.pre-commit-config.yaml

```diff
@@ -54,7 +54,7 @@ repos:
   #  additional_dependencies: [flake8-bugbear]
 
 - repo: https://github.com/pre-commit/mirrors-mypy
-  rev: 'v0.910'  # Use the sha / tag you want to point at
+  rev: 'v0.981'  # Use the sha / tag you want to point at
```

Comment:
Had to bump it to fix `pre-commit` for newer Python versions. https://github.com/python/mypy/issues/13627